src/Dialogs/Weather/SkysightDialog.cpp

```diff
@@ -82,6 +82,37 @@ class SkysightListItemRenderer
   std::shared_ptr<Skysight> skysight;
 };
 
+//TODO: Could extend this to extract Skysight data at point and display on list (see NOAAListRenderer::Draw(2)
+void
+SkysightListItemRenderer::Draw(Canvas &canvas, const PixelRect rc, unsigned i) {
+  const ComputerSettings &settings = CommonInterface::GetComputerSettings();
+  SkysightActiveMetric m = SkysightActiveMetric(skysight->GetActiveMetric(i));
+
+  tstring first_row = tstring(m.metric->name);
+  if (skysight->displayed_metric == m.metric->id.c_str())
+    first_row += " [ACTIVE]";
+
+  StaticString<256> second_row;
+
+  if (m.updating) {
+    second_row.Format("%s", _("Updating..."));
+  } else {
+    if (!m.from || !m.to || !m.mtime) {
+      second_row.Format("%s", _("No data. Press \"Update\" to update."));
+    } else {
+      uint64_t elapsed = BrokenDateTime::NowUTC().ToUnixTimeUTC() - m.mtime;
+
+      second_row.Format(_("_Data from %s to %s. Updated %s ago"), 
```

Comment:
Why does this string start with an underscore?
Will this be translatable to all languages? (grammar!)

Reply:
No reason to start with an underscore. Will remove.
Skysight chart is available every 30 minutes. So second_row is "Data from 10:30 to 11:00. Updated 1 sec ago" for example. I may translate to Japanese as "10:30 kara 11:00 no de-ta. 1 byo maeni koushin". Is there any rule for localization?